### PR TITLE
Added lecture 41. Teaching the learner to have a radio button checked for guarantee of account type.

### PR DESCRIPTION
**Pull request by NiharPhansalkar**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by jeremylt** [comment]
This does not have an associated issue. Can you explain why you think these changes are needed?

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+id: 6338628e24620bf3848eb082
+title: Step 42
+challengeType: 0
+dashedName: step-42
+---
+
+# --description--
+
+step 41 instructions
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md:10]: This is incomplete

Comment (NiharPhansalkar) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md:10]: Step 42 has not been added by me. I only added Step 41 in the editor.

**Comment by NiharPhansalkar** [comment]
The issue I have tried to solve is the following 
https://github.com/freeCodeCamp/freeCodeCamp/issues/47456

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md (hunk 1 above)
Comment (jeremylt) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md:10]: You have mistakenly added two steps here

**Comment by jeremylt** [comment]
> The issue I have tried to solve is the following
> #47456

You should add the issue to the PR description. Also, reading the contribution guidelines (the box you checked) will tell you how to title the PR.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md (hunk 1 above)
Comment (NiharPhansalkar) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md:10]: It is incrementing the steps by 2, even though I am adding only one step. Can you please tell me what I can do to fix this? I worked on it last night, and I cannot seem to fix it.

Comment (jeremylt) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/6338628e24620bf3848eb082.md:10]: I am not an expert with the challenge editor. All I know is that you added two additional files, as shown in the list of changes.

**Comment by NiharPhansalkar** [comment]
I am extremely sorry. I will re-do the entire process.

**Pull request closed without merging**